context.h

```diff
@@ -29,6 +29,7 @@
 struct Email;
 struct EmailList;
 struct NotifyCallback;
+extern struct ContextList contexts;
```

Comment:
I think we usually name globals CameCase?

Reply:
OK, changed in new commit  1c77574.